JKBTumpalan/evtf_project
# Feature/pages rps classifier

# Ticket
- None

# Changes
- Rock, paper, scissors classifier API endpoint
- Page behavior and state management

# Dependencies
None

# Notes
- None

# Test Plan
1. Pull branch
2. change working directory to **_/evtf_project_**
3. `docker-compose up --build`
- [ ] Check if the rps classifier is working both via upload and via image link input
- [ ] Check if the input area updates whenever image link or upload button is selected
- [ ] Check if the image preview is present for image upload input 
- [ ] Check if the RPS labels show after prediction
- [ ] Check if the RPS prediction is correct.
- [ ] Check if the buttons are protected while uploading
- [ ] Check if the buttons are disabled if there are empty input fields
